Sources/Commands/Workspace.swift

```diff
@@ -468,6 +448,17 @@ public class Workspace {
             }
         }
     }
+
+    /// Load a manifest which is guaranteed to load because it has been loaded successfully previously.
+    /// A failure here indicates the manifest was changed somehow outside of SwiftPM.
+    // FIXME: Eventually get these manifests from a persistence cache.
+    func loadManifest(packagePath path: AbsolutePath, baseURL: String, version: Version?) throws -> Manifest {
+        do {
+            return try manifestLoader.load(packagePath: path, baseURL: baseURL, version: version)
+        } catch {
+            throw WorkspaceOperationError.unexpectedManifestLoadingError("Manifest at \(path.asString) unexpectedly failed to load. Try a clean build.")
```

Comment:
I wrote clean build here considering clean will wipe out the checkouts (but keep the repositories)